shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -519,6 +519,27 @@ public static WireCommand readFrom(DataInput in, int length) throws IOException
         }
     }
 
+    @Data
+    public static final class CloseAppend implements WireCommand {
+        final WireCommandType type = WireCommandType.CLOSE_APPEND;
+        final UUID writerId;
+        final String segment;
```

Comment:
Do we actually need to pass the segment here?

Reply:
yes, see the setupSegments table;  Entry is a pair of <writer id, segment>.

Reply:
@tkaitchuck 
yes  you are right..
I have done the changes setupSegments -> setupWriters with writer id as key;
No, this enable us to remove the segment name from flush and close wire command.